radio/src/main_avr.cpp

```diff
@@ -129,6 +132,9 @@ void perMain()
   const char *warn = s_warning;
   uint8_t menu = s_menu_count;
 
+#if defined(LCD_ST7920)
+  if(0==lcdstate){//No need to redraw until lcdRefresh_ST7920(0) below completely refreshes the display.
```

Comment:
It's difficult to read, would you add a definition, something like IS_LCD_REFRESH_ALLOWED() that would remove the #ifdef here?